swift-generator/src/main/java/com/facebook/swift/generator/ConstantRenderer.java

```diff
@@ -200,7 +201,7 @@ private String renderIntegerConstant(BaseType baseType, ConstInteger value)
 
             case I64:
                 return Long.toString(value.value()) + "L";
-
+                
```

Comment:
All the other case statements had a newline between cases - default didn't for some reason